neon-mmd/websurfx
# Make Gitpod work by fixing the config file

## What does this PR do?

This PR adds several fixes, as suggested by @alamin655 in PR #122.

## Why is this change important?

This change is essential as it adds numerous fixes which enables the Gitpod to work properly.

## How to test this PR locally?

To test this PR, launch the project with Gitpod and see if everything gets setup as expected.

Gitpod link: https://gitpod.io/#https://github.com/neon-mmd/websurfx

## Author's checklist

- [x] Add fixes to the config file for Gitpod.
- [x] Bump the `Cargo.toml` version.

## Related issues

#118
